index.js

```diff
@@ -69,74 +87,96 @@ module.exports = {
     if (this.shouldIncludePolyfill()) {
       this.importPolyfill(app);
     }
-  }
-};
+  },
 
-function getAddonOptions(addonContext) {
-  var baseOptions = (addonContext.parent && addonContext.parent.options) || (addonContext.app && addonContext.app.options);
-  return baseOptions && baseOptions.babel || {};
-}
-
-function getBabelOptions(addonContext) {
-  var options = clone(getAddonOptions(addonContext));
-  var ui = addonContext.ui;
-
-  // pass a console object that wraps the addon's `UI` object
-  options.console = {
-    log: function(message) {
-      // fallback needed for support of ember-cli < 2.2.0
-      if (ui.writeInfoLine) {
-        ui.writeInfoLine(message);
-      } else {
-        ui.writeLine(message, 'INFO');
-      }
-    },
-
-    warn: function(message) {
-      // fallback needed for support of ember-cli < 2.2.0
-      if (ui.writeWarnLine) {
-        ui.writeWarnLine(message);
-      } else {
-        ui.writeLine(message, 'WARN');
-      }
-    },
-
-    error: function(message) {
-      // fallback needed for support of ember-cli < 2.2.0
-      if (ui.writeError) {
-        ui.writeError(message);
-      } else {
-        ui.writeLine(message, 'ERROR');
+  _getAddonOptions: function() {
+    return (this.parent && this.parent.options) || (this.app && this.app.options) || {};
```

Comment:
@nathanhammond that is pretty much the original code and not modified by me (see https://github.com/babel/ember-cli-babel/blob/b9a0dc232f4038bf6c95de51f71b04faff08406d/index.js#L76)